Whiskey/en-US/about_Whiskey_Tasks.help.txt

```diff
@@ -0,0 +1,66 @@
+TOPIC
+    about_Whiskey_Tasks
+    
+SHORT DESCRIPTION
+    This topic lists all Whiskey tasks and the global task properties 
+    (i.e. properties available on every task).
+
+LONG DESCRIPTION
+
+    ## Tasks
+
+    Whiskey has the following tasks:
+
+    * `CopyFile`: copies files.
+    * `Delete`: deletes files.
+    * `Exec`: runs an executable/program.
+    * `GetPowerShellModule`: installs a PowerShell module.
+    * `MSBuild`: runs MSBuild, the Microsoft build engine.
+    * `NodeLicenseChecker`: runs license checker, a Node module/command that 
+      scans and reports on Node module license metadata.
+    * `NodeNspCheck`: runs NSP, a Node security scanner.
+    * `NpmConfig`: runs the Node `npm config` command.
+    * `NpmInstall`: runs the Node `npm install` command.
+    * `NpmPrune`: runs the Node `npm prune` command.
+    * `NpmRunScript`: runs the Node `npm run-script` command.
+    * `NuGetPack`: runs the `nuget.exe pack` command.
+    * `NuGetPush`: runs the `nuget.exe push` command.
+    * `NuGetRestore`: runs the `nuget.exe restore` command.
+    * `NUnit2`: tests .NET assemblies with NUnit 2.
+    * `NUnit3`: tests .NET assemblies with NUnit 3.
+    * `Pester3`: runs PowerShell tests wth Pester 3.
+    * `Pester4`: runs PowerShell tests with Pester 4.
+    * `Pipeline`: runs pipelines defined in the current whiskey.yml file.
+    * `PowerShell`: executes a PowerShell script.
+    * `ProGetUniversalPackage`: creates a universal ProGet package.
+    * `PublishBitbucketServerTag`: tags a repository in Bitbucket Server.
+    * `PublishBuildMasterPackage`: creates and deploys a package in BuildMaster.
+    * `PublishNodeModule`: publishes a Node module (i.e. runs the `npm publish`
+      command).
+    * `PublishPowerShellModule`: publishes a PowerShell module.
+    * `PublishProGetAsset`: publishes a file to a ProGet asset directory.
+    * `PublishProGetUniversalPackage`: publishes a universal ProGet package to
+      ProGet.
+    * `SetVariable`: creates a variable that you can use in your whiskey.yml
+      file.
+    * `TaskDefaults`: sets default task properties.
+
+    ## Common Task Properties
+
+    Every Whiskey task has these properties:
+
+    * `OnlyBy`/`ExceptBy`: controls if a task runs if run by a developer or

```

Comment:
`controls if a task runs if run` might flow better with something like `controls if a task should run when a build is executed`.